core/unit_test/TestViewLayoutStrideAssignment.hpp

```diff
@@ -617,6 +617,171 @@ TEST(TEST_CATEGORY, view_layoutstride_right_to_layoutright_assignment) {
   }
 }
 
+TEST(TEST_CATEGORY, view_layoutstride_right_to_layoutleft_assignable) {
```

Comment:
There's *way* too much duplicated code between this and the `view_layoutstride_left_to_layoutright_assignable` test below.

Reply:
This still needs to be addressed.